lib/loader.js

```diff
@@ -16,10 +16,14 @@ module.exports = function(content, map) {
 	var root = query.root;
 	var moduleMode = query.modules || query.module;
 	var camelCaseKeys = query.camelCase || query.camelcase;
+	var sourceMap = query.sourceMap || false;
 	var resolve = createResolver(query.alias);
 
-	if(map !== null && typeof map !== "string") {
+	if(sourceMap && map !== null && typeof map !== "string") {
 		map = JSON.stringify(map);
+	} else {
+        // Some loaders (example `"postcss-loader": "1.x.x"`) always generates source map, we should remove it
+		map = null;
```

Comment:
@evilebottnawi - As a general rule of thumb, you aren't going to get a hold of Tobias via Github notifications and more so within the `webpack-contrib` organization.

In the contrib org, the org-maintainers or admins will contact Tobias ( or another core member ) only when a problem or change requires a deeper level of knowledge than we collectively have. The `webpack` members are focusing on webpack core development and maintenance, we are focused on everything under contrib.

Within the scope of this pull request, @jhnns has done most of the recent performance testing and it's his input that is really required in this case.